pulsar-io/kafka/src/main/java/org/apache/pulsar/io/kafka/KafkaSourceConfig.java

```diff
@@ -69,6 +69,11 @@
         help =
             "The timeout used to detect failures when using Kafka's group management facilities.")
     private long sessionTimeoutMs = 30000L;
+    @FieldDoc(
+        defaultValue = "3000",
+        help =
+            "The interval between heartbeats to the consumer when using Kafka's group management facilities.")
```

Comment:
can you highlight that hearbeat interval need to be less than session timeout ms?

Reply:
Thanks! I pushed 6e497c1d30bc7de2780822da69c1cfb3ffc86cc0 , please review again.